src/App.js

```diff
@@ -427,6 +442,7 @@ ${INDENT[this.IL]})`;
         if (classesUsed.indexOf(inheritsFrom) === -1) {
           classesUsed.push(inheritsFrom);
         }
+        addUsedClass(inheritsFrom, allClasses);
```

Comment:
Didn't have second level inheritance accounted for (ie: `NewestThing > Thing` and then `Thing > OldestThing`), but this covers that.